modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -134,14 +134,15 @@ public void processOpts() {
 
     @Override
     public String getTypeDeclaration(Schema p) {
+        String oasType = getSchemaType(p);
         if (ModelUtils.isFileSchema(p)) {
             return "Blob";
         } else if (ModelUtils.isBinarySchema(p)) {
             return "Blob";
         } else if (ModelUtils.isDateSchema(p)) {
-            return "Date";
+            return typeMapping.containsKey("Date") ? typeMapping.get("Date") : "Date";
```

Comment:
not sure this is correct.
According to `DefaultCodegen#getPrimitiveType()` implementation, `"Date"` is not a valid primitive type. Instead, `"date"` is (more precisely, `SchemaTypeUtil.DATE_FORMAT`).

Again, the AbstractTsClient already includes a reasonable mapping for it (maps to "string": `typeMapping.put("date", "string");`). So having it in the child TS codegens seems redundant